week-6/Homework/mandatory/1-alarmclock/style.css

```diff
@@ -1,15 +1,85 @@
+body{
+  display: flex;
+  font-family: 'Rye', cursive;
+  font-size: xx-large;
+  background-color:#f06588 ;
+}
 .centre {
   position: fixed;
+  text-align: center;
   top: 50%;
   left: 50%;
   -webkit-transform: translate(-50%, -50%);
   transform: translate(-50%, -50%);
 }
+.settime_font{
+  display: flex;
+  justify-content: space-evenly;
+}
 
 #alarmSet {
-  margin: 20px;
+  margin-bottom:5%;
+  margin-right: 5%;
 }
 
-h1 {
-  text-align: center;
+#timeRemaining{
+  margin-bottom: 15%;
+  font-family: 'Rye', cursive;
+  font-size: 200%;
```

Comment:
Because `font-size: 200%;` is not inside a media query, it will only take effect for the screen widths you have left out in your media queries. 

The media queries you've used are:
`max-width:800px` --> from 0 to 800px
`(min-width:800px) and (max-width:1500px)` --> from 800px to 1500px

So what is left is 1500px and above. Which means `font-size: 200%;` will be applied to 1500px and above. Is this what you intended? If so, that's fine.

The "mobile first" approach, which is a very good practice, does things the other way around:

1.  First, you define some CSS for mobile devices, **without using a media query**. 

2. Then, you write some media queries starting at a certain size, for example 800px. This would mean using `(**min**-width:800px)` (not max-width). 

3. Don't set an upper limit in your media queries: only use `min-width: ....px` and don't set a max-width.

4. Keep your styles in ascending order: styles for smaller screens at the top, followed by medium screens, then the  biggest at the end.  This will allow you to keep better control of your styles, and nothing will overwrite anything else.